octue/cloud/deployment/google/cloud_run.py

```diff
@@ -6,21 +6,16 @@
 
 from octue.cloud.pub_sub.service import Service
 from octue.exceptions import MissingServiceID
-from octue.logging_handlers import apply_log_handler
 from octue.resources.service_backends import GCPPubSubBackend
 from octue.runner import Runner
 
 
 logger = logging.getLogger(__name__)
-apply_log_handler(logger, log_level=logging.INFO)
-
+app = Flask(__name__)
 
 DEPLOYMENT_CONFIGURATION_PATH = "deployment_configuration.json"
 
 
-app = Flask(__name__)
-
```

Comment:
Why has this switched? Better to keep constants sat at the top by convention, no?

Reply:
I wasn't sure whether it's better to have constants first (e.g. `DEPLOYMENT_CONFIGURATION_PATH`) and global objects after, or the reverse. What do you think?

Reply:
I'd always expect to see constants first, by generalish convention in most langauses (except for some with explicit and weird globals like fortran's shared modules stuff which had to be defined at the top)